--- xbmc-18.0a1-Leia/tools/depends/native/TexturePacker/Makefile~	2018-03-04 11:21:28.000000000 +0200
+++ xbmc-18.0a1-Leia/tools/depends/native/TexturePacker/Makefile	2018-04-01 16:45:56.499884518 +0300
@@ -10,19 +10,11 @@
 
 ifeq ($(NATIVEPLATFORM),)
   PLATFORM = native
-  EXTRA_CONFIGURE = --enable-static
 else
   PLATFORM = $(NATIVEPLATFORM)
   DEPS += ../../Makefile.include
 endif
 
-ifeq ($(NATIVE_OS), linux)
-  EXTRA_CONFIGURE = --enable-static
-endif
-ifeq ($(NATIVE_OS), android)
-  EXTRA_CONFIGURE = --enable-static
-endif
-
 ifeq ($(CMAKE_SOURCE_DIR),)
   CMAKE_SOURCE_DIR = $(ROOT_DIR)/../../../..
 endif
