--- xbmc-17.6-Krypton/tools/depends/native/TexturePacker/Makefile~	2017-11-14 18:55:01.000000000 +0200
+++ xbmc-17.6-Krypton/tools/depends/native/TexturePacker/Makefile	2017-12-24 18:05:39.101238711 +0200
@@ -9,18 +9,10 @@
 
 ifeq ($(NATIVEPLATFORM),)
   PLATFORM = native
-  EXTRA_CONFIGURE = --enable-static
 else
   PLATFORM = $(NATIVEPLATFORM)
 endif
 
-ifeq ($(NATIVE_OS), linux)
-  EXTRA_CONFIGURE = --enable-static
-endif
-ifeq ($(NATIVE_OS), android)
-  EXTRA_CONFIGURE = --enable-static
-endif
-
 ifeq ($(CORE_SOURCE_DIR),)
   CORE_SOURCE_DIR = $(ROOT_DIR)/../../../..
 endif
