--- xbmc-16.1-Jarvis/xbmc/filesystem/FTPParse.cpp.orig	2016-04-24 08:48:30.000000000 +0200
+++ xbmc-16.1-Jarvis/xbmc/filesystem/FTPParse.cpp	2017-12-11 17:11:03.079955624 +0100
@@ -34,6 +34,8 @@
 #include <cmath>
 #include "FTPParse.h"
 
+using std::string;
+
 CFTPParse::CFTPParse()
 {
   m_flagtrycwd = 0;
--- xbmc-16.1-Jarvis/lib/cximage-6.0/CxImage/ximadsp.cpp.orig	2016-04-24 08:48:30.000000000 +0200
+++ xbmc-16.1-Jarvis/lib/cximage-6.0/CxImage/ximadsp.cpp	2017-12-11 18:34:55.026753161 +0100
@@ -3,12 +3,14 @@
  * CxImage version 6.0.0 02/Feb/2008
  */
 
+#if CXIMAGE_SUPPORT_DSP
+// include <queue> before any local headers, as they define min/max as macros
+#include <queue>
+
 #include "ximage.h"
 
 #include "ximaiter.h"
 
-#if CXIMAGE_SUPPORT_DSP
-
 ////////////////////////////////////////////////////////////////////////////////
 /**
  * Converts the image to B&W.
@@ -3477,7 +3479,6 @@
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-#include <queue>
 ////////////////////////////////////////////////////////////////////////////////
 /**
  * Flood Fill
